DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -106,6 +106,8 @@
       </nav>
       <div class="visual-separator"></div>
     </header>
+    
+    @RenderSection("NavBreadcrumbs", required: false)

```

Comment:
This should be merged to master really. But as it's such a small change it should be fine to handle the merge conflicts of doing it on both branches. FYI @benjimarshall (as putting it on master is part of your ticket).

Reply:
Thanks, I'll see where we are when I've finished HEEDLS-286 🙂